espresso-jshell/README.md

```diff
@@ -1,51 +1,56 @@
 # GraalVM Demos: Native jshell and Espresso
 
-This demo showcases the integration between [GraalVM Native Image](https://www.graalvm.org/reference-manual/native-image/) and [Java-on-Truffle (Espresso)](https://www.graalvm.org/reference-manual/java-on-truffle/).
-You will build a native executable of `jshell`, that executes dynamically generated bytecode on Espresso.
-This hybrid mode achieves instant startup, beating the vanilla `jshell` in both: time to the first interaction and time to evaluate a simple expression.
+This demo showcases the integration between [GraalVM Native Image](https://www.graalvm.org/reference-manual/native-image/) and [Java-on-Truffle (Espresso)](https://www.graalvm.org/reference-manual/java-on-truffle/).  
+It builds a native image of `jshell`, that executes the dynamically generated bytecodes on Espresso. This hybrid mode achieves instant startup, beating the vanilla `jshell` in both: time to the first interaction and time to evaluate a simple expression.
 
-JShell is a Java read-eval-print loop tool first introduced in the JDK 9, but this demo also shows how to run `jshell` on Java 8.
+JShell is a Java read-eval-print loop tool first introduced in Java 9, this demo also allows to run `jshell` on Java 8.
```

Comment:
```suggestion
JShell is a Java read-eval-print loop tool first introduced in Java 9, this demo also allows running `jshell` on Java 8.
```

Reply:
I manually fixes this nits in the latest version.